myuw/templates/handlebars/card/accounts/tuition.html

```diff
@@ -51,37 +51,43 @@ <h4 class="card-badge-label">Account Credit</h4>
                     </li>
                 </ul>
             </div>
-        {{ else }}  <!-- else, there is a balance -->
-            <div class="card-badge-container">
+        {{else}}  <!-- else, there is a balance -->
+            <div class="card-badge-container tuition">
 	        <ul class="card_list">
 		    <li>
 	                <div class="card-badge clearfix">
 	                    <div class="pull-left">
 	                        <h4 class="card-badge-label">You owe</h4>
 	                    </div>
-			    <div class="pull-right">
+						<div class="pull-right">
 	                        <span class="card-badge-value text-danger">${{ tuition_accbalance }}</span>
-			        &nbsp;
-	                        <span class="card-badge-action pull-right"><a href="http://f2.washington.edu/fm/sfs/tuition/payment" target="_blank" data-linklabel="Make Tuition Payment">Make payment</a></span>
-	                    </div>
+	            		</div>
 	                </div>
 		    </li>
 
 	            {{#if tuition_date}}
 	            <li>
-	                <div class="card-badge clearfix">
+	                <div class="card-badge clearfix no-top-margin">
 	                    <div class="pull-left">
 	                        <h4 class="card-badge-label">Tuition due </h4>
 	                    </div>
-			    <div class="pull-right">
+						<div class="pull-right">
 	                        <span class="card-badge-value" aria-hidden="true">{{ toFriendlyDate tuition_due }}</span>
-			        <span class="sr-only">{{ toFriendlyDateVerbose tuition_due }}</span>
+							<span class="sr-only">{{ toFriendlyDateVerbose tuition_due }}</span>
 			        &nbsp;
 	                        <span class="card-badge-action pull-right">in {{ tuition_date_offset }} days</span>
 	                    </div>
 	                </div>
 		    </li>
 	            {{/if}}
+		    
```

Comment:
per our discussion, include the button into the list item that it belongs to (PCE or regular tuition).